services/ui-src/src/utils/other/mask.test.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  convertToCommaSeparatedString,
+  isCustomMask,
+  isNumberStringMaskable,
+  maskValue,
+} from "utils";
+
+const commaSeparatedMaskAcceptableTestCases = [
+  { test: "0....0123", expected: "0.01" },
+  { test: "000000123", expected: "123.00" },
+  { test: "123", expected: "123.00" },
+  { test: "123.00", expected: "123.00" },
+  { test: ".05000000000000", expected: "0.05" },
+  { test: ".05", expected: "0.05" },
+  { test: ".5", expected: "0.50" },
+  { test: "0.05", expected: "0.05" },
+  { test: "1,234.00", expected: "1,234.00" },
+  { test: "1,234", expected: "1,234.00" },
+  { test: "1234", expected: "1,234.00" },
+  { test: "1,234.00", expected: "1,234.00" },
+  { test: "1,234", expected: "1,234.00" },
+  { test: "100,000,000", expected: "100,000,000.00" },
+  { test: "100000000", expected: "100,000,000.00" },
+  {
+    test: "Technically a wrong input that validation would pick up but would actually still work with 1 number",
+    expected: "1.00",
+  },
```

Comment:
Would remove this one and make it so the number masks only operate on strings composed entirely of 0123456789.,. Even though the number mask can handle it we don't really need to test that since our number field validation won't be allowing any through.
```suggestion
```